index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="<link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="header">
+    <h1 class="Mylogo"><a href="#">My Logo</a></h1>
+    <nav>
+      <a href="#"><img src="images/Union.png" alt="Union"></a>
+    </nav>
+  </header>
+  <main>
+    <section class="headline">  
+      <h2>I’m Henary
+        Glad to see you!</h2>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <div class="social">
+          <p>LET’S CONNECT</p>
+          <ul class="social-icon">
+            <li class="social-item"><a href="https://twitter.com/HenaryKhuraijam"><img src="images/twitter.png" alt="twitter"></a></li>
+            <li class="social-item"><a href="https://www.linkedin.com/in/henary-khuraijam-50487317a/"><img src="images/linkedin.png" alt="linkedin"></a></li>
+            <li class="social-item"><a href="https://www.linkedin.com/in/henary-khuraijam-50487317a/"><img src="images/linkedin.png" alt="linkedin"></a></li>
```

Comment:
- [ ] You have added an extra `LinkedIn` icon for this section. It doesn't match the given template. Kindly fix this issue.
  - ![image](https://user-images.githubusercontent.com/60083437/219147593-e71132ae-9e3e-45f6-80d4-c324b54258a4.png)